features/step_definitions/customer_booking_request_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^a location enabled for online booking$/) do
+  Locations.online_booking_location_uids = %w(ac7112c3-e3cf-45cd-a8ff-9ba827b8e7ef)
```

Comment:
Good observation. I'm OK with being explicit about this for now. I'm providing the alternative in the `no locations are enabled for online booking` so we're at least covered in the places this will affect for now. If it becomes untenable in the future we'll address it then.